helpdesk_mgmt/data/helpdesk_data.xml

```diff
@@ -16,7 +16,9 @@
             <field
                 name="email_cc"
             >${not object.partner_id and object.partner_email or ''|safe},</field>
-            <field name="subject">The ticket ${object.number} has been closed.</field>
+            <field
+                name="subject"
+            >The ticket "${object.display_name}" has been closed.</field>
```

Comment:
I don't see that this full information should be put here.

Reply:
That's not something for the subject of the mail

Reply:
Ok, it is fair to not accept this change, at the end is a matter opinable so, @mariadforgeflow I think we should keep it as it is, we can change the template in our installation